lld/MachO/Config.h

```diff
@@ -256,6 +256,8 @@ struct Configuration {
   llvm::MachO::PlatformType platform() const {
     return platformInfo.target.Platform;
   }
+
+  bool deduplicateSymbolStrings = true;
```

Comment:
Probably makes sense to put this next to
https://github.com/llvm/llvm-project/blob/2a51a0d39a659feeeee57b6d1d768bf08d378c5e/lld/MachO/Config.h#L145